engine/src/paragrafattr.cpp

```diff
@@ -1225,6 +1225,8 @@ void MCParagraph::computerects(int32_t x, int32_t y, int32_t p_layout_width, uin
 		case kMCParagraphTextAlignRight:
 			t_offset = p_layout_width - t_total_width;
 			break;
+		default:
+			MCUnreachableReturn();
```

Comment:
@montegoulding I don't think this is enough, unfortunately.  In Release builds, this will be compiled out, so won't catch the error.  There's nothing in the field's implementation that provides compile-time assurance that the return value of `gettextalign()` will be a member of the (anonymous) text alignment enumeration.  I believe that the "right" fix here is to leave the switch how it is, and initialise `int32_t t_offset = 0;`. Possible alternatives would be to explicitly make the left alignment case the `default:` case, or set `t_offset = 0;` just before `MCUnreachabelReturn()`.